### ISSUE TITLE
isSupported does not work for anything other than NDEF on iOS

### ISSUE THREAD

**Issue by thecalvinchan** [whitedogg13/react-native-nfc-manager#224]
seems like `isSupported` never got updated on iOS from v1.

`IsoDep` is returning `false` for `isSupported`, even though it is supported

Code below shows we are only checking for `Ndef` or a blank string

```
RCT_EXPORT_METHOD(isSupported: (NSString *)tech callback:(nonnull RCTResponseSenderBlock)callback)
{
    if (isSupported()) {
        // iOS only supports Ndef starting from iOS 11.0 (on iPhone 7 onwards)
        if ([tech isEqualToString:@""] || [tech isEqualToString:@"Ndef"]) {
            callback(@[[NSNull null], @YES]);
            return;
        }
    }

    callback(@[[NSNull null], @NO]);
}
```

**Comment by whitedogg13** [comment]
@thecalvinchan thanks for reporting, this should be fixed in next release.

**Comment by whitedogg13** [comment]
Fixed in latest v2.0.2, thanks!
